src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -82,14 +102,30 @@ public Tenant save(UUID consortiumId, Tenant tenantDto) {
 
     checkTenantNotExistsAndConsortiumExistsOrThrow(consortiumId, tenantDto.getId());
     Tenant savedTenant = saveTenant(consortiumId, tenantDto);
-
+    User shadowAdminUser = userTenantService.prepareShadowUser(adminUserId, currentTenantContext.getTenantId(), currentTenantContext);
+    executeAfterTransactionCommits(() ->
     runInFolioContext(createFolioExecutionContextForTenant(tenantDto.getId(), currentTenantContext, folioMetadata),
-      () -> configurationClient.saveConfiguration(createConsortiaConfigurationBody(centralTenantId)));
+      () -> {
+        createShadowAdminUserWithPermissions(shadowAdminUser);
```

Comment:
this method just creates shadow admin with permissions, but doesn't assign to affiliation

Reply:
fixed